code/modules/client/preferences_toggles.dm

```diff
@@ -104,15 +104,6 @@
 		src << "You will no longer prayer sounds."
 	feedback_add_details("admin_verb", "PSounds")
 
-/client/verb/togglePRs()
-	set name = "Show/Hide Pull Request Announcements"
-	set category = "Preferences"
-	set desc = "Toggles receiving a notification when new pull requests are created."
-	prefs.chat_toggles ^= CHAT_PULLR
-	prefs.save_preferences()
-	src << "You will [(prefs.chat_toggles & CHAT_PULLR) ? "now" : "no longer"] see new pull request announcements."
-	feedback_add_details("admin_verb","TPullR") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
-

```

Comment:
It was the longest verb name, so it was forcing the tab to only have 1(maybe two on widescreens) column because of its name. This allows the pref tab to have two or three columns.

You can toggle it in the game preference menu anyway. No need to have it here, its unlikely somebody would want to quickly toggle it